sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/FluxAutoRenewTest.java

```diff
@@ -0,0 +1,431 @@
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.messaging.servicebus.implementation.LockContainer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import reactor.core.Disposable;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+import reactor.test.publisher.TestPublisher;
+
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Function;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * Unit test for {@link FluxAutoRenew}.
+ */
+public class FluxAutoRenewTest {
+
+    private static final UUID LOCK_TOKEN_UUID = UUID.randomUUID();
+    private static final String LOCK_TOKEN_STRING = LOCK_TOKEN_UUID.toString();
+    private static final Duration MAX_AUTO_LOCK_RENEW_DURATION = Duration.ofSeconds(6);
+    private static final Duration DISABLE_AUTO_LOCK_RENEW_DURATION = Duration.ofSeconds(0);
+
+    private final ClientLogger logger = new ClientLogger(FluxAutoRenewTest.class);
+
+    private final ServiceBusReceivedMessage message = new ServiceBusReceivedMessage("Some Data".getBytes());
+    private final TestPublisher<ServiceBusReceivedMessage> messagesPublisher = TestPublisher.create();
+    private final Flux<? extends ServiceBusReceivedMessage> messageSource = messagesPublisher.flux();
+    private Function<String, Mono<OffsetDateTime>> renewalFunction;
+
+    private OffsetDateTime lockedUntil;
+
+    @Mock
+    LockContainer<LockRenewalOperation> messageLockContainer;
+
+    @BeforeAll
+    static void beforeAll() {
+        StepVerifier.setDefaultTimeout(Duration.ofSeconds(30));
+    }
+
+    @AfterAll
+    static void afterAll() {
+        StepVerifier.resetDefaultTimeout();
+    }
+
+    @BeforeEach
+    void setup() {
+        MockitoAnnotations.initMocks(this);
+        lockedUntil = OffsetDateTime.now().plusSeconds(2);
+        message.setLockToken(LOCK_TOKEN_UUID);
+        message.setLockedUntil(lockedUntil);
+        renewalFunction = (lockToken) -> Mono.just(OffsetDateTime.now().plusSeconds(1));
```

Comment:
Should be consistent about adding `(` around your lambdas. there are several down there that don't have it.

Reply:
Same with other instances.